### ISSUE TITLE
X11 mining not working

### ISSUE THREAD

**Issue by BlackMouse** [cbuchner1/ccminer#7]
X11 Mining isn’t working for me, I’ve tried on different pools and blocks are detected, kh/s is displayed but it doesn’t tell me that blocks are accepted…

Edit: I'm using a gtx460, is it related to that ?

![ccminer_fail](https://cloud.githubusercontent.com/assets/1719250/2938172/75f32a90-d8f5-11e3-8af3-a90712833fa0.PNG)

**Comment by chigley** [comment]
+1, on a GTX 570.

**Comment by KBomba** [comment]
https://bitcointalk.org/index.php?topic=167229.msg6669157#msg6669157

**Comment by chigley** [comment]
Awesome, thanks! I thought my hash rate was high :smile:

**Comment by BlackMouse** [comment]
I started to mine heavy coin to buy darkcoin later, I get 0.008 btc/day with that

(I'm using ccminer 0.5)

**Comment by KBomba** [comment]
You could try out the V1.0 I've been able to compile for compute capability 2.1. It has X11 support too https://bitcointalk.org/index.php?topic=167229.msg6676746#msg6676746 
If you can, compile it for yourself ;) For the 570, compute 2.0 I was unable to get it working with just that (HTTP errors), but it was only tested on compute 2.1 devices, maybe it works for you, chigley? Mail me if you want a compiled version.

**Comment by BlackMouse** [comment]
Great, thank you very much, I'll try it when i'll get back from work :)

2014-05-12 1:19 GMT+02:00 KBomba notifications@github.com:

**Comment by BlackMouse** [comment]
It works perfectly, thank you :D

Envoyé avec AquaMail pour Android
http://www.aqua-mail.com

**Comment by KBomba** [comment]
Just interested, how much are you getting with your gtx460 @ x11?

**Comment by BlackMouse** [comment]
800 kh/s (~0.008 btc/day)
It doesn't prevent me from using my computer correctly
I don't recommend mining with cpu at the same time, it ruins performances

2014-05-13 13:01 GMT+02:00 KBomba notifications@github.com:

**Comment by callmedoodman** [comment]
Im trying to build a version for compute 1.2.  Have some old quadro fx laptops that could be put to use.  Any help would be greatly appreciated.  Not a programmer.  Very lost so far.

**Comment by cbuchner1** [comment]
A lot of the algorithms in X11 require hardware features of Compute 3.0
devices. Only the simplest hashing algorithms would still build on Compute
1.2. Time to upgrade to a Maxwell based card, I suppose (GTX 750 or GTX
960).

But. GPU mining is pretty much dead (barely covers electricity).

Christian

2015-05-26 22:10 GMT+02:00 callmedoodman notifications@github.com:

**Comment by auroracoin** [comment]
Hi Christian. That you for the nvidia start with cudaminer.
11 750ti's gets me 1.35 btc a month at $75 USD in electricity . Which equals $244.00 USD a month . :-)

**Comment by cbuchner1** [comment]
Yes there are profitable niches, but they stop being profitable once people
learn about them ;)

2015-05-28 17:35 GMT+02:00 auroracoin notifications@github.com:

**Comment by auroracoin** [comment]
Yes true. But the trick is to find them and keep up with them. :-)
